src/index.js

```
@@ -3,14 +3,15 @@ import { config } from "dotenv";
 import cors from "cors";
 import mongoose from "mongoose";
 import morgan from "morgan";
+import helmet from "helmet";
 import fs from 'fs';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)